### PR TITLE
I beautified the profile page real nice like

### LINKED ISSUE #123: Profile page is ugly
It wastes a lot of space. We can probably combine both of those options in tabs or one long page.
screenshot from 2018-04-15 15-49-55

### PR DESCRIPTION
**Pull request by nchurchill**
I have added in a tab structure to contain the information for enrolling in institutions,courses,and programs so we no longer just have the two buttons floating about

fixes #123

### CODE REVIEW

File: src/Athena/Views/Student/Profile.cshtml [hunk 1]
```diff
@@ -1,17 +1,61 @@
...
+
+    <script type="text/javascript" src="@Url.Content("~/studentSetup.js")"></script>
+    <script type="text/javascript">
+        $(function() {
+            athena.studentSetup.init('@(student.Id)');
+        });
+    </script>
+    <script type="text/javascript" src="@Url.Content("~/completedCourses.js")"></script>
+    <script type="text/javascript">
+            athena.completedCourses.init("@(student.Id)");
```
[the remark this answers is not in the document]
Comment (nchurchill) [reply src/Athena/Views/Student/Profile.cshtml]: fixed

File: src/Athena/Views/Student/Profile.cshtml [hunk 2]
```diff
@@ -1,17 +1,61 @@
...
     </div>
-</div>
\ No newline at end of file
+</div>
+
+
+
+
+@section scripts
+    {
```
[the remark this answers is not in the document]
Comment (nchurchill) [reply src/Athena/Views/Student/Profile.cshtml]: done

File: src/Athena/Views/Student/UserSetup.cshtml [hunk 3]
```diff
@@ -1,77 +0,0 @@
-﻿@model string
```
[the remark this answers is not in the document]
Comment (nchurchill) [reply src/Athena/Views/Student/UserSetup.cshtml]: done

File: src/Athena/Controllers/AccountController.cs [hunk 4]
```diff
@@ -155,7 +155,7 @@ public IActionResult LoginWithExternalProvider(string provider, string returnUrl
                         
                         _log.Information("{user} created an account via {provider}", info.Principal.Identity.Name, info.ProviderDisplayName);
 
-                        return RedirectToAction(nameof(StudentController.UserSetup), "Student", new {returnUrl});
+                        return RedirectToAction(nameof(StudentController.Profile), "Student", new {returnUrl});
```
Comment (nlowe) [inline src/Athena/Controllers/AccountController.cs]: The profile page doesn't have a `returnUrl` parameter, you can remove it.

**Pull request merged**